res/values/strings.xml

```diff
@@ -327,7 +332,7 @@
     <string name="login_smtp_server">SMTP server</string>
     <string name="login_smtp_port">SMTP port</string>
     <string name="login_smtp_security">SMTP security</string>
-    <string name="login_auth_method">Authorization method</string>
+    <string name="login_auth_method">Authentication method</string>
```

Comment:
what is the reason for this chage?

options are "auto" and "oauth2" - the latter is about "authorization",  https://oauth.net/2/ . so, this is hard. if in doubt, i would just leave it unchanged.

Reply:
Changes was undone.